packages/at_secondary_server/pubspec.yaml

```diff
@@ -34,6 +34,13 @@ dependencies:
   yaml: 3.1.2
   logging: 1.2.0
 
+dependency_overrides:
```

Comment:
if the PR in at_commons is merged, can you please public and replace with published version

Reply:
replaced in [d451144](https://github.com/atsign-foundation/at_server/pull/1536/commits/d451144e477d4ace76e786f5f1b58755d9ee58da)